pir/cpp/client_test.cpp

```diff
@@ -59,10 +60,12 @@ class PIRClientTest : public ::testing::Test {
 
 TEST_F(PIRClientTest, TestCreateRequest) {
   const size_t desired_index = 5;
+  const vector<size_t> indexes = {desired_index};
```

Comment:
nit: indices is the more correct term here. So long as you're consistent as you are here, it should be fine.